plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlClient.java

```diff
@@ -202,7 +202,6 @@ public void testConvertLike()
                 createTestingTypeAnalyzer(PLANNER_CONTEXT),
                 TypeProvider.viewOf(Map.of(new Symbol("c_varchar"), VARCHAR_COLUMN.getColumnType())),
                 PLANNER_CONTEXT))
-                // Currently, LIKE with ESCAPE isn't converted to ConnectorExpression. Update the test when it gets converted.
-                .isEmpty();
+                .isPresent();
```

Comment:
The change is necessary in this commit (as it adds translation), but isn't a good one (the test code looks pointless).
Add a TODO comment in the first commit to indicate this is just a placeholder change.